easybuild/easyconfigs/p/pangolin/pangolin-3.1.10-foss-2020b.eb

```diff
@@ -0,0 +1,75 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# Based on a EC provided by Paul Jähn
+# uploaded by J. Sassmannshausen
+
+easyblock = 'PythonBundle'
+
+name = 'pangolin'
+version = '3.1.10'
+
+homepage = 'https://cov-lineages.org/pangolin.html'
+description = "Software package for assigning SARS-CoV-2 genome sequences to global lineages."
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Biopython', '1.78'),
+    ('gofasta', '0.0.5', '', True),
+    ('minimap2', '2.20'),
+    ('Pysam', '0.16.0.1'),
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('snakemake', '6.1.0'),
+    ('UShER', '0.4.1'),
+    ('scikit-learn', '0.23.2'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+github_account = 'cov-lineages'
+exts_list = [
+    ('pangolin', '3.1.11', {
+        'source_urls': ['https://github.com/%(github_account)s/%(name)s/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['ce64a3ef84de5b11d5c9f21f76341cae23c0fabbf46cc3c1b7b244701b36efbc'],
+    }),
+    ('pangoLEARN', '2021-08-09', {
+        'modulename': 'pangoLEARN',
+        'source_urls': ['https://github.com/%(github_account)s/%(name)s/archive'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['ece55bc37b3e35dbdff2432fb8b9ea76fa53fe993394a425d1df67147484a81e'],
+    }),
+    ('scorpio', '0.3.12', {
+        'source_urls': ['https://github.com/%(github_account)s/%(name)s/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['48390ed1c1a79bb5f801083042cdeddee060e2277f40c293fd30384882aced81'],
+    }),
+    ('constellations', '0.0.13', {
+        'source_urls': ['https://github.com/%(github_account)s/%name)s/archive'],
+        'sources': [{'download_filename': 'v%(version).tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
```

Comment:
```suggestion
        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
```

Reply:
Thanks for finding them! I simply did not spot them. (I do need new glasses, not a joke). I have corrected them.
By the way, is there  a way to avoid these repeating lines?

Reply:
Converted to draft as we are waiting to have `UShER` merged.

Reply:
> By the way, is there a way to avoid these repeating lines?

You can likely put them into `exts_default_options` (dict that is merged with this dict) although I'm not fully sure it works with the templates, but please try.
Also use `SOURCE_TAR_GZ` as the `filename` and `GITHUB_SOURCE` as the source_url, see https://docs.easybuild.io/en/latest/version-specific/easyconfig_templates.html

> Converted to draft as we are waiting to have UShER merged.

If that is the only thing then simply mention it in the PR description like: `- [ ] Requires #13708`